pkg/migrations/migrations.go

```diff
@@ -398,3 +402,43 @@ func optInToDiagnosticsStatReporting(ctx context.Context, r runner) error {
 	}
 	return err
 }
+
+// repopulateViewDeps ensures that each view V that depends on a table
+// T depends on all columns in T. (#17269)
+func repopulateViewDeps(ctx context.Context, r runner) error {
```

Comment:
I can't figure a way to create an invalid view descriptor in package `migration` without exporting a bunch of stuff from `sql` and `sqlbase`. What would you suggest?